### ISSUE TITLE
how to use plugins in vanilla javascript 

### ISSUE THREAD

**Issue by ctxwing** [naver/egjs-view3d#55]
## Description
I'm just coding in  vanilla javascript.
as for plugins, I've followed link below: 
https://naver.github.io/egjs-view3d/docs/options/miscellaneous/plugins

it says using import but not available in  vanilla `javascript.`
at present I am using 
`    <script src="js/view3d.pkgd.min.js"></script>
    <link rel="stylesheet" href="css/view3d-bundle.min.css">

    <script src="js/jquery-3.5.1.min.js"></script>
`

is it impossible in  vanilla javascript or am I lost ?
or using mixing  jQuery and egjs-view3d  can cause problem?

in short,  how to use plugins in vanilla javascript ?

thanks in advance for good solution..

**Comment by WoodNeck** [comment]
Hello @ctxwing!
If you're importing View3D with a script tag, every component in View3D is exported like this:
```js
View3D.XXX
```

So, try using it like this:
```js
const viewer = new View3D("#el", {
  plugins: [new View3D.LoadingBar()]
})
```

Or...

```js
const viewer = new View3D("#el");

viewer.addPlugins(new View3D.LoadingBar());
```

**Comment by ctxwing** [comment]
@WoodNeck 
Thanks for quick response.. it works !
as for me, 'viewer.loadPlugins(new View3D.LoadingBar());' works fine.
(instead of  'viewer.addPlugins(new View3D.LoadingBar());' does not work)

would you please update this in  docs for it? if it is possible. thanks again.

**Comment by WoodNeck** [comment]
@ctxwing That's my mistake, `loadPlugins` is right!
- https://naver.github.io/egjs-view3d/docs/plugins
- https://naver.github.io/egjs-view3d/docs/api/View3D#loadPlugins
